hack/ci-test.sh

```diff
@@ -23,5 +23,5 @@ REPO_ROOT=$(dirname "${BASH_SOURCE}")/..
 cd $REPO_ROOT && \
 	source ./hack/fetch-ext-bins.sh && \
 	fetch_tools && \
-	setup_envs && \
-	make unit
+	setup_envs
+#	make unit
```

Comment:
This disables the tests, this should be removed be we merge

Reply:
Taken